internal/analytics/event_utils.go

```diff
@@ -0,0 +1,30 @@
+package analytics
```

Comment:
Can you rename this file to be `events.go` since "utils" doesn't add any meaning?